src/docdb/registerDocDBCommands.ts

```diff
@@ -46,7 +48,13 @@ export function registerDocDBCommands(): void {
         }
         const childNode = await node.createChild(context);
         await commands.executeCommand("cosmosDB.openStoredProcedure", childNode);
-
+    });
+    registerCommandWithTreeNodeUnwrapping('cosmosDB.createDocDBTrigger', async (context: IActionContext, node?: DocDBTriggersTreeItem) => {
```

Comment:
I prefer keeping this command here for now. To me the organization still makes sense since all the commands registered in this file are all DocDB related (aka. CosmosDB NoSQL). I want to move all the implementation details out of this file so that this file contains merely a list of "register command" calls.